pdns/lua-record.cc

```diff
@@ -356,11 +368,15 @@ static bool getLatLon(const std::string& ip, string& loc)
 
 static ComboAddress pickclosest(const ComboAddress& bestwho, const vector<ComboAddress>& wips)
 {
+  if (wips.empty()) {
+    throw std::invalid_argument("The IP list cannot be empty");
+  }
   map<double,vector<ComboAddress> > ranked;
   double wlat=0, wlon=0;
   getLatLon(bestwho.toString(), wlat, wlon);
   //        cout<<"bestwho "<<wlat<<", "<<wlon<<endl;
   vector<string> ret;
+  g_log<<Logger::Debug<< __FILE__ << ":" << __LINE__<< " " << __func__ << " wips.size()="<< wips.size() <<endl;
```

Comment:
Please wrap these in `DLOG(..)` (unless they will be removed before merge, of course!)

Reply:
(I see now they do get removed later)